<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1292</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1292</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="displayVideo" name="videoWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="frontButton">
              <property name="toolTip">
               <string>W</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="wifiVehicleImages.qrc">
                <normaloff>:/wifiVehicleImages/image/12W453331340-G1C8.png</normaloff>:/wifiVehicleImages/image/12W453331340-G1C8.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="leftButton">
              <property name="toolTip">
               <string>A</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="wifiVehicleImages.qrc">
                <normaloff>:/wifiVehicleImages/image/12W45332bZ-D1406.png</normaloff>:/wifiVehicleImages/image/12W45332bZ-D1406.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rightButton">
              <property name="toolTip">
               <string>D</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="wifiVehicleImages.qrc">
                <normaloff>:/wifiVehicleImages/image/12W453330V0-F40K.png</normaloff>:/wifiVehicleImages/image/12W453330V0-F40K.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="backButton">
              <property name="toolTip">
               <string>S</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="wifiVehicleImages.qrc">
                <normaloff>:/wifiVehicleImages/image/12W453330430-EBR.png</normaloff>:/wifiVehicleImages/image/12W453330430-EBR.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="connectButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="wifiVehicleImages.qrc">
                <normaloff>:/wifiVehicleImages/image/connect.png</normaloff>:/wifiVehicleImages/image/connect.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="disconnectButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="wifiVehicleImages.qrc">
                <normaloff>:/wifiVehicleImages/image/disconnect.png</normaloff>:/wifiVehicleImages/image/disconnect.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Status</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="styleSheet">
                    <string notr="true">font: 11pt &quot;Consolas&quot;;</string>
                   </property>
                   <property name="text">
                    <string>Moving status:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>18</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="moveState">
                   <property name="styleSheet">
                    <string notr="true">font: 11pt &quot;Consolas&quot;;
color: rgb(255, 10, 0)</string>
                   </property>
                   <property name="text">
                    <string>Stopped</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="styleSheet">
                    <string notr="true">font: 11pt &quot;Consolas&quot;;</string>
                   </property>
                   <property name="text">
                    <string>Control method:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>28</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="controlState">
                   <property name="styleSheet">
                    <string notr="true">font: 11pt &quot;Consolas&quot;;
color: rgb(255, 10, 0)</string>
                   </property>
                   <property name="text">
                    <string>Unknown</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="styleSheet">
                    <string notr="true">font: 11pt &quot;Consolas&quot;;</string>
                   </property>
                   <property name="text">
                    <string>Net status:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="netState">
                   <property name="styleSheet">
                    <string notr="true">font: 11pt &quot;Consolas&quot;;
color: rgb(255, 10, 0)</string>
                   </property>
                   <property name="text">
                    <string>Disconnected</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="readTempButton">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Read 
Temperature</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="tempLcdNumber"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QSlider" name="pwmMotorSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="pwmMotorLcdNumber"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="buzzerButton">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Buzzer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exitButton">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Exit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="startCamButton">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="wifiVehicleImages.qrc">
              <normaloff>:/wifiVehicleImages/image/chinaz_PNG17.png</normaloff>:/wifiVehicleImages/image/chinaz_PNG17.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopCamButton">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="wifiVehicleImages.qrc">
              <normaloff>:/wifiVehicleImages/image/stop_cam.png</normaloff>:/wifiVehicleImages/image/stop_cam.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="capFrameButton">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="wifiVehicleImages.qrc">
              <normaloff>:/wifiVehicleImages/image/photo.png</normaloff>:/wifiVehicleImages/image/photo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSTART">
    <property name="title">
     <string>Start(开始)</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOPTION">
    <property name="title">
     <string>Options(配置)</string>
    </property>
    <addaction name="actionIP_Setup_IP"/>
    <addaction name="separator"/>
    <addaction name="actionPicture_directory_setup"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(帮助)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSTART"/>
   <addaction name="menuOPTION"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="wifiVehicleImages.qrc">
     <normaloff>:/wifiVehicleImages/image/connect.png</normaloff>:/wifiVehicleImages/image/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect(连接)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="wifiVehicleImages.qrc">
     <normaloff>:/wifiVehicleImages/image/stop.png</normaloff>:/wifiVehicleImages/image/stop.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect(断开)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="wifiVehicleImages.qrc">
     <normaloff>:/wifiVehicleImages/image/gnome-shutdown.png</normaloff>:/wifiVehicleImages/image/gnome-shutdown.png</iconset>
   </property>
   <property name="text">
    <string>Exit(退出)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionIP_Setup_IP">
   <property name="icon">
    <iconset resource="wifiVehicleImages.qrc">
     <normaloff>:/wifiVehicleImages/image/stock_file-properites.png</normaloff>:/wifiVehicleImages/image/stock_file-properites.png</iconset>
   </property>
   <property name="text">
    <string>IP Setup(IP端口设置)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="wifiVehicleImages.qrc">
     <normaloff>:/wifiVehicleImages/image/gtk-about.png</normaloff>:/wifiVehicleImages/image/gtk-about.png</iconset>
   </property>
   <property name="text">
    <string>About(关于)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionPicture_directory_setup">
   <property name="icon">
    <iconset resource="wifiVehicleImages.qrc">
     <normaloff>:/wifiVehicleImages/image/dir.png</normaloff>:/wifiVehicleImages/image/dir.png</iconset>
   </property>
   <property name="text">
    <string>Saved picture setup (保存图片路径)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>displayVideo</class>
   <extends>QWidget</extends>
   <header location="global">displayvideo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="wifiVehicleImages.qrc"/>
 </resources>
 <connections/>
</ui>
